libs/core/src/lib/api/billing.service.ts

```diff
@@ -90,6 +92,19 @@ export class BillingService {
     );
   }
 
+  getStripeSubscription(): Observable<StripeAccountSubscription | null> {
+    return this.getSubscription().pipe(
+      map((subscription) => {
+        if (!(subscription as unknown as any).provider) {
+          return null;
+        } else {
+          return (subscription as AccountSubscription).subscription as StripeAccountSubscription;
+        }
+      }),
+      catchError(() => of(null)),
+    );
+  }
```

Comment:
Actually with the typing from `getSubscription`, there's a lot of cast that you don't need here:

```suggestion
  getStripeSubscription(): Observable<StripeAccountSubscription | null> {
    return this.getSubscription().pipe(
      map((subscription) => {
        if (!subscription.provider) {
          return subscription;
        } else {
          return subscription.subscription;
        }
      }),
      catchError(() => of(null)),
    );
  }
```